src/pages/code-examples.tsx

```diff
@@ -112,7 +112,7 @@ export const getStaticProps: GetStaticProps = async () => {
                     text: tag.name,
                     url: tag.slug + '/code-examples',
                 },
-                imageString: languageToLogo[tag.name],
+                imageString: languageToLogo[tag.name] || null,
```

Comment:
why is the logic different for code-examples and tutorials alone?

Reply:
also lot of code looks similar between code-examples and tutorials, can we please move that into common functionality ?

Reply:
Also not in this PR , but why are we using mock data for videos?

Reply:
That's a good point, it should be separated. I will make that change. And for the videos, i think that was just a mistake on my end, forgot to add the logic for getting the first 3 content pieces.